app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -91,4 +91,17 @@
     }
   }
 
+  .back-to-content {
+    @include core-19;
+    display: block;
+    padding-bottom: $gutter-half;
+
+    &:before {
+      content: "\2191";
+    }
+
+    @media print {
```

Comment:
Static also has a `dont-print` class we could use for this:
https://github.com/alphagov/static/blob/31f79a61ca5535ec805c02ff690616823cc32ad9/app/assets/stylesheets/helpers/_print-base.scss#L85